machine.go

```diff
@@ -151,14 +142,9 @@ type Config struct {
 	// firecracker. If not provided, the default signals will be used.
 	ForwardSignals []os.Signal
 
-	// SeccompLevel specifies whether seccomp filters should be installed and how
-	// restrictive they should be. Possible values are:
-	//
-	//	0 : (default): disabled.
-	//	1 : basic filtering. This prohibits syscalls not whitelisted by Firecracker.
-	//	2 : advanced filtering. This adds further checks on some of the
-	//			parameters of the allowed syscalls.
-	SeccompLevel SeccompLevelValue
+	// SeccompCfg specifies whether seccomp filters should be installed and how
+	// restrictive they should be.
+	SeccompCfg SeccompConfig
```

Comment:
How about calling this field just `Seccomp`? We are sadly not really consistent about adding `Cfg` on the fields, but `config.Seccomp.Enabled = true` looks better than `config.SeccompCfg.Enabled = true`.

Reply:
updated.